.github/workflows/triage-issue.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Label issues
         uses: andymckay/labeler@1.0.3
```

Comment:
I just noticed something.  The `- reopened` needs to be removed on line # 5.

```diff
name: Triage Issue
on:
  issues:
    types:
-      - reopened
      - opened
```

Reply:
Changes made